src/main/java/dev/meyi/bn/commands/BazaarNotifierCommand.java

```diff
@@ -218,7 +226,7 @@ public void processCommand(ICommandSender ics, String[] args) {
           player.addChatMessage(new ChatComponentText(BazaarNotifier.prefix + EnumChatFormatting.RED
               + "All module locations have been reset and the order list has been emptied."));
         } else if (args[1].equalsIgnoreCase("orders") && args.length == 2) {
-          BazaarNotifier.orders = Defaults.DEFAULT_ORDERS_LAYOUT();
+          BazaarNotifier.orders = new LinkedList<>();

```

Comment:
```suggestion
          BazaarNotifier.orders.clear()
```

For some reason I'm confused as to why we can't use this instead.